src/modules/requests/requests.repository.ts

```diff
@@ -59,19 +54,7 @@ export class RequestsRepository {
     return result;
   }
 
-  updateStatus(id: string, status: number) {
-    return this.request.update(id, { status });
-  }
-
-  updateItem(id: string, updated: UpdateItem) {
-    return this.request.update(id, updated);
-  }
-
-  updateFilePath(id: string, updated: UpdateFilename) {
-    return this.request.update(id, updated);
-  }
-
-  updateNotes(id: string, updated: UpdateNotes) {
+  update(id: string, updated: Update) {
```

Comment:
minor naming variable kang 😄 
```suggestion
  update(id: string, update: Update) {
```

Reply:
siapp 😄